OpenRA.Mods.Common/Traits/Player/ModularBot.cs

```diff
@@ -25,15 +25,16 @@ public sealed class ModularBotInfo : TraitInfo, IBotInfo
 		[Desc("Internal id for this bot.")]
 		public readonly string Type = null;
 
+		[TranslationReference]
 		[Desc("Human-readable name this bot uses.")]
-		public readonly string Name = "Unnamed Bot";
+		public readonly string Name = null;
```

Comment:
Worth applying this change in `DummyBot` as well? Which appears to be used in Fort Lonestar.

Reply:
There is a limitation of the map rules translation not showing in the lobby in this case, so I reverted that change.

Reply:
This sounds like another case that will need additional plumbing - since there is surely going to be other cases (e.g. mission descriptions) that will need this.

The cleanest solution would be to extend `MapPreview` and the resource center API to load the map specified translations.

Reply:
This is fixed by translating deeper down the chain.

Reply:
We still need the `[TranslationReference]` in `DummyBotInfo` I believe?

Reply:
It still does not work. I won't be able to solve it in this pull request.

Reply:
The `MapPreview` changes will also be needed for any custom map defined bots.